src/domains/User/Commands/LoginUser.command.ts

```diff
@@ -0,0 +1,10 @@
+import { ICommand } from '@nestjs/cqrs';
+import { JwtService } from '@nestjs/jwt';
+
+export class LoginUserCommand implements ICommand {
+  constructor(
+    public username: string,
+    public password: string,
+    public jwtService: JwtService,
```

Comment:
I wanted to sign within the aggregate. But JwtService (AuthModule inits it) can't be injected within UserModule, because AuthModule already imports UserModule so if UserModule imports AuthM there is circular dependency.

So now I'm passing the jwtService through many layers.
But I'm now realising there is no need to store the jwt in the User? 
In auth.service, if the commandBus returns the user, all went alright and auth service can sign jwt and return it